packages/serverpod/lib/src/server/log_manager/log_manager.dart

```diff
@@ -209,7 +203,42 @@ class LogManager {
   /// logged before calling this method. This method can be called
   /// asynchronously.
   @internal
-  Future<void> logMessage(Session session, MessageLogEntry entry) async {
+  Future<void> logMessage(
+    Session session, {
+    required String endpointName,
+    required String messageName,
+    required int messageId,
+    required Duration duration,
+    required String? error,
+    required StackTrace? stackTrace,
+  }) async {
+    var executionTime = duration.inMicroseconds / (1000 * 1000.0);
+
+    var slow = executionTime >=
+        settings.getLogSettingsForSession(session).slowSessionDuration;
+
+    var shouldLog = _shouldLogMessage(
```

Comment:
Question: are the shouldLog conditions the same for the different logging reasons?

Reply:
No!